src/js/tracks/text-track-settings.js

```diff
@@ -321,17 +323,12 @@ class TextTrackSettings extends Component {
    * @method restoreSettings
    */
   restoreSettings() {
-    let err;
     let values;
 
     try {
-      [err, values] = safeParseTuple(window.localStorage.getItem('vjs-text-track-settings'));
```

Comment:
I still think it's nice to use safeParseTuple. I think the reason why this is wrapped in a try/catch is to make sure that the call to localStorage doesn't break.